lib/ansible/plugins/callback/__init__.py

```diff
@@ -555,11 +555,32 @@ def v2_runner_on_skipped(self, result: TaskResult) -> None:
             host = result._host.get_name()
             self.runner_on_skipped(host, self._get_item_label(getattr(result._result, 'results', {})))
 
-    def v2_runner_on_unreachable(self, result):
+    def v2_runner_on_unreachable(self, result: TaskResult) -> None:
+        """Get details about a task that Ansible could not perform because it could not reach a
+        targeted host, then process the details as required by the callback (output, profiling,
+        logging, notifications, etc.)
+
+        Customization note: For more information about the attributes and methods of the
+        TaskResult class, see lib/ansible/executor/task_result.py.
```

Comment:
This doesn't seem to align with the purpose/concept of docstrings. Docstrings explain what something is/does.

For internal motivation/explanation notes, code comments are better.

Reply:
I rewrote the comment so it aligns with your comment, "Docstrings explain what something is/does," but I left it in the docstring so that it appears in the VS Code and PyCharm hover-hints for all classes based on the `CallbackBase` class, instead of replicating it in the code-comments of each callback.

Reply:
Marking this thread as resolved.